Dockerfile

```diff
@@ -0,0 +1,115 @@
+# Copyright (C) 2020 Bosch Software Innovations GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+FROM frolvlad/alpine-java:jdk8-slim AS build
+
+COPY . /usr/local/src/ort
+
+WORKDIR /usr/local/src/ort
+
+RUN apk add --no-cache \
+    # Required for Node.js to build the reporter-web-app.
+    libstdc++ \
+    # Required to allow to download via a proxy with a self-signed certificate.
+    ca-certificates coreutils openssl && \
+    scripts/import_proxy_certs.sh
+
+RUN scripts/set_gradle_proxy.sh && \
+    ./gradlew --no-daemon :cli:distTar
+
+FROM openjdk:11-jre-slim-sid
+
+ENV \
```

Comment:
Did you consider using `ARG` instead of `ENV` for e.g. all these package manager versions as these are needed only for building the image?

Reply:
Good idea, will do.

Reply:
The downside of `ARG` is that in contrast to `ENV` it does not support [setting multiple values with a single line](https://github.com/moby/moby/issues/35950), i.e. you have to use one `ARG` per argument, and as `ARG` (like `ENV`) creates a new layer, we end up with *many* layers. So I'll revert that change again.